nemo_skills/inference/eval/hilbench_assets/litellm_shim/sitecustomize.py

```
@@ -0,0 +1,142 @@
+    """Read a field from a message that may be a dict OR an object (litellm Message/pydantic)."""
+    if isinstance(m, dict):
+        return m.get(key)
+    return getattr(m, key, None)
+
+
+def _msg_set_content(m, value):
+    if isinstance(m, dict):
+        m["content"] = value
+    else:
+        try:
+            setattr(m, "content", value)
```

Ruff (0.15.20)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)